modules/dynamic-plugins/ref-rh-compatible-plugins.adoc

```diff
@@ -19,10 +19,10 @@ The following Technology Preview plugins are not preinstalled and must be instal
 |`https://access.redhat.com/downloads/content/480/ver=2.4/rhel---9/2.4/x86_64/product-software[@ansible/plugin-scaffolder-backend-module-backstage-rhaap]` |1.0.0 |
 
 
-|Orchestrator Frontend|`https://www.npmjs.com/package/@janus-idp/backstage-plugin-orchestrator[@janus-idp/backstage-plugin-orchestrator]` |1.22.6| https://www.parodos.dev/1.2-rc/docs/[Learn more]
+|Orchestrator Frontend|`https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator[@red-hat-developer-hub/backstage-plugin-orchestrator]` |1.22.6| https://www.parodos.dev/1.2-rc/docs/[Learn more]
```

Comment:
latest is 2.5.0 but not sure if that's compatible with RHDH 1.4?

@batzionb  WDYT we should have here?

```suggestion
|Orchestrator Frontend|`https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator[@red-hat-developer-hub/backstage-plugin-orchestrator]` |2.5.0| https://www.parodos.dev/1.2-rc/docs/[Learn more]
```